### PR TITLE
Add friend page design

### PR DESCRIPTION
**Pull request by mayankvishwakarma012**
**Linked Work Items**
-https://internship-splitwise.atlassian.net/jira/software/projects/IS/issues/IS-62

**Checklist**
- [x] Code adheres to **[Git and Pull Request Guidelines]**
- [x] Code adheres to **[Angular Guidelines]**
- [x] Code adheres to **[Front-end Checklist]**

### CODE REVIEW

**Comment by ravipromact** [comment]
1. Only first letter should be capitalized for commits

File: src/app/friends/add-edit-friends/add-edit-friends.component.html [hunk 1]
```diff
@@ -1 +1,41 @@
...
+        <div class="mb-3">
+            <label for="friend-name" class="form-label fs-5">Friends Email: </label>
+            <input type="email" class="form-control" id="friend-name" placeholder="Enter Group Name" formControlName="friendName">
+        </div>
+
+        <!-- message  -->
+        <div class="mb-3">
+            <label for="Textarea" class="form-label fs-5">Message: </label>
+            <div class="form-floating">
+                <textarea class="form-control" placeholder="Leave a comment here" id="Textarea" style="height: 100px" formControlName="message"></textarea>
```
Comment (ravipromact) [inline src/app/friends/add-edit-friends/add-edit-friends.component.html]: 1. Remove inline style and use css instead
2. Lowercase 'T' from id Textarea

File: src/app/friends/view-friends-list/view-friends-list.component.ts [hunk 2]
```diff
@@ -7,4 +9,29 @@ import { Component } from '@angular/core';
...
 export class ViewFriendsListComponent {
 
+
+  friends = ['Ram', 'Harry', 'Raj'];
+
+  currentUrl !: string;
+  isFriendActive : boolean = false;
+  friendName: any;
+  constructor(public router : Router, public route: ActivatedRoute){
+    this.route.params.subscribe(params => {
```
Comment (ravipromact) [inline src/app/friends/view-friends-list/view-friends-list.component.ts]: Add comment above to inform what this.route.params.subscribe does

File: src/app/friends/view-friends-list/view-friends-list.component.ts [hunk 3]
```diff
@@ -7,4 +9,29 @@ import { Component } from '@angular/core';
...
+      this.friendName = params['name'];
+    });
+
+    this.router.events
```
Comment (ravipromact) [inline src/app/friends/view-friends-list/view-friends-list.component.ts]: Add comments above

File: src/app/friends/add-edit-friends/add-edit-friends.component.html (hunk 1 above)
Comment (mayankvishwakarma012) [reply src/app/friends/add-edit-friends/add-edit-friends.component.html]: I changed the id of textarea and removed the inline style.

File: src/app/friends/view-friends-list/view-friends-list.component.ts (hunk 2 above)
Comment (mayankvishwakarma012) [reply src/app/friends/view-friends-list/view-friends-list.component.ts]: Added comments.

File: src/app/friends/view-friends-list/view-friends-list.component.ts (hunk 3 above)
Comment (mayankvishwakarma012) [reply src/app/friends/view-friends-list/view-friends-list.component.ts]: Added comments.

**Review by ravipromact — APPROVED** [review]

**Pull request merged**